### PR TITLE
Added masternode worth

### PR DESCRIPTION
**Pull request by dkbay**
Added new stat "MasternodeWorth"

### CODE REVIEW

**Comment by JSponaugle** [comment]
This will mess up the current Styling of the site.

File: resources/views/layout/statsbar2.blade.php [hunk 1]
```diff
@@ -5,11 +5,11 @@
             <div class="col-md-1"></div>
             <div class="col-md-2 bardata">
                 <div class="bardatatitle">Avg. Blocks Awarded</div>
-                <div class="bardatadata"><span style="color:#FF8B41">{!! number_format($avgblocks,'2','.',',') !!}</span></div>
+                <div class="bardatadata">{!! number_format($avgblocks,'2','.',',') !!}</div>
```
Comment (JSponaugle) [inline resources/views/layout/statsbar2.blade.php]: This is Breaking Current Layout

File: resources/views/layout/statsbar2.blade.php [hunk 2]
```diff
@@ -5,11 +5,11 @@
...
             </div>
             <div class="col-md-2 bardata">
                 <div class="bardatatitle">Avg. Reward Freqency</div>
-                <div class="bardatadata"><span style="font-size:70%;color:#008080">{!! number_format($avgrewardfreq,'2','.',',') !!} hrs</span></div>
+                <div class="bardatadata">{!! number_format($avgrewardfreq,'2','.',',') !!} <span style="font-size:70%;color:teal">hrs</span></div>
```
Comment (JSponaugle) [inline resources/views/layout/statsbar2.blade.php]: Break Current Layout

File: resources/views/layout/statsbar2.blade.php [hunk 3]
```diff
@@ -34,19 +34,19 @@
             </div>
             <div class="col-md-2 bardata">
                 <div class="bardatatitle">24hr Block Count</div>
-                <div class="bardatadata"><span style="color:#FF8B41">{!! $block24hour !!}</span></div>
+                <div class="bardatadata"><span style="color:teal">{!! $block24hour !!}</span></div>
```
Comment (JSponaugle) [inline resources/views/layout/statsbar2.blade.php]: CSS is Wrong

File: resources/views/layout/statsbar2.blade.php [hunk 4]
```diff
@@ -34,19 +34,19 @@
...
             </div>
             <div class="col-md-2 bardata">
                 <div class="bardatatitle">Avg. Block Time</div>
-                <div class="bardatadata"><span style="color:#008080">{!! number_format($avgblocktime,'1','.','') !!}</span></div>
+                <div class="bardatadata"><span style="teal">{!! number_format($avgblocktime,'1','.','') !!}</span></div>
```
Comment (JSponaugle) [inline resources/views/layout/statsbar2.blade.php]: CSS Wrong

File: resources/views/layout/statsbar2.blade.php [hunk 5]
```diff
@@ -34,19 +34,19 @@
...
             </div>
             <div class="col-md-2 bardata">
                 <div class="bardatatitle">Masternode Block Reward</div>
                 <div class="bardatadata"><span style="color:#1D82AD">{!! $blockreward / 2 !!}</span></div>
             </div>
             <div class="col-md-2 bardata">
                 <div class="bardatatitle">Days Until Reward Drop</div>
-                <div class="bardatadata"><span style="color:#008080">{!! $daytilldrop !!}</span></div>
+                <div class="bardatadata"><span style="color:teal">{!! $daytilldrop !!}</span></div>
```
Comment (JSponaugle) [inline resources/views/layout/statsbar2.blade.php]: CSS Wrong

**Comment by dkbay** [comment]
Fixed stuff

**Pull request merged**